test/verify/ruff.toml

```diff
@@ -1,4 +1,5 @@
 extend = "../common/ruff.toml"
 ignore = [
     "B023",     # Function definition does not bind loop variable
+    "PT",       # HACK: ignore all of PT as our tasks container ruff does not know about PT027 yet
```

Comment:
I don't like this.  What happens when we ignore just "PT027"?

Reply:
Ah, the mommit message has a better motivation: "Our integration tests are not using pytest but unittest.TestCase so we want to silence these rules." That makes sense to me, but then the comment shouldn't say that we only want to ignore "PT027".

Reply:
Yes, adjusted the comment.